ansible/roles/samza-jobs-additional-config/tasks/main.yml

```diff
@@ -1,12 +1,29 @@
 ---
 - name: Create directory for additional config
-  file: path={{object_denormalization_additional_config_dir}} owner=hduser group=hadoop recurse=yes state=directory
+  file:
+    path: "{{ object_denormalization_additional_config_dir }}"
```

Comment:
Object Denormalization job is no longer used and can be removed.

Reply:
removed the role as requested.